sklearn/ensemble/base.py

```diff
@@ -88,8 +88,7 @@ def __init__(self, base_estimator, n_estimators=10,
 
         # Don't instantiate estimators now! Parameters of base_estimator might
         # still change. Eg., when grid-searching with the nested object syntax.
-        # This needs to be filled by the derived classes.
-        self.estimators_ = []
```

Comment:
I wonder if we need to deprecate this, since it is mutable and some ensembles support a `warm_start` that could be interpreted as pre-populating this list. However, doing the deprecation right could be messy. I vote for documenting this change in particular in what's new instead.